crates/astria-sequencer-relayer/local.env.example

```diff
@@ -19,8 +19,12 @@ ASTRIA_SEQUENCER_RELAYER_GAS_LIMIT=1000000
 # requesting a new block from sequencer.
 ASTRIA_SEQUENCER_RELAYER_BLOCK_TIME=1000
 
+# Set to true to only relay blocks proposed by the validator key in `ASTRIA_SEQUENCER_RELAYER_VALIDATOR_KEY_FILE`.
+ASTRIA_SEQUENCER_RELAYER_DISABLE_RELAY_ALL=false
+
 # The path to the file storing the cometbft validator key.
 # Used to sign data before submitting it to celestia.
+# Ignored if `ASTRIA_SEQUENCER_RELAYER_DISABLE_RELAY_ALL=false`.
```

Comment:
When I made the suggestion to express validator construction in another review comment I realized that this naming confused me.

Can we switch the logic and call this `ASTRIA_SEQUENCER_RELAYER_RELAY_ONLY_BLOCKS_MATCHING_VALIDATOR_KEY`? or `*_RELAY_ONLY_VALIDATOR_KEY_BLOCKS`?

I know it's wordy, but easier to grok.